docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,170 @@
+
+```js
+Excel.run(function (context) {
+	var docProperties = context.workbook.properties;
+	docProperties.author = "Alex";
+	return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+You can also define custom properties. The `DocumentProperties` object has `custom`: a map defining key-value pairs for user-defined properties. The following example shows how to create a custom "Introduction" property with the value "Hello", then retrieve it.
+
+```js
+Excel.run(function (context) {
+	var customDocProperties = context.workbook.properties.custom;
+	customDocProperties.add("Introduction", "Hello");
+	return context.sync();
+}).catch(errorHandlerFunction);
+
+[...]
+
+Excel.run(function (context) {
+	var customDocProperties = context.workbook.properties.custom;
+	var customProperty = customDocProperties.getItem("Introduction");
+	customProperty.load("key, value");
+
+	return context.sync().then(function() {
+        console.log("Custom key  : " + customProperty.key); // "Introduction"
+        console.log("Custom value : " + customProperty.value); // "Hello"
+	});
+}).catch(errorHandlerFunction);
+```
+
+## Document settings
+
+A workbook's settings are similar to the collection of custom properties. The difference is settings are unique to a single Excel file and add-in pairing, whereas properties are solely connected to the file. The following example shows how to create and access a setting.
+
+```js
+Excel.run(function (context) {
+	var settings = context.workbook.settings;
+	settings.add("NeedsReview", true);
+	var needsReview = settings.getItem("NeedsReview");
+	needsReview.load("value");
+
+	return context.sync().then(function() {
+		console.log("Workbook needs review : " + needsReview.value);
+	});
+}).catch(errorHandlerFunction);
+```
+
+## Calculations
+
+By default, Excel recalculates formula results whenever a referenced cell is changed. Your add-in's performance may benefit from adjusting this calculation behavior. The application object has a `calculationMode` property of type `CalculationMode`. It can be set to the following values:
+
+ - `automatic`: The default recalculation behavior where Excel calculates new formula results every time the relevant data is changed.
+ - `automaticExceptTables`: Same as `automatic`, except any changes made to values in tables are ignored.
+ - `manual`: Calculations only occur when the user or add-in requests them.
+ 
+Application objects provide a method to force an immediate recalculation. `Application.calculate(calculationType)` starts a manual recalculation based on the provided `CalculationType`. The following values can be provided:
```

Comment:
- "Application objects provide..." -> "The Application object provides..."
- "based on the provided `CalculationType`" -> ""based on the specified `calculationType`"
- '"The following values can be provided:" -> "The following values can be specified:"